src/atoms/language.ts

```diff
@@ -3,7 +3,8 @@ import { atomWithStorage } from "jotai/utils";
 import { Language } from "../types";
 import strings from "../localization/strings";
 
-const languageAtomBase = atomWithStorage<Language>("language", strings.getLanguage() as Language);
+const defaultLanguage = navigator.language === 'fi' ? strings.getLanguage() : 'en-gb';
```

Comment:
Where possible we use double quotes (") rather than single quotes. Change all usages of single quotes to double quotes